internal/api/rbac/utils.go

```diff
@@ -118,7 +124,12 @@ func (r *cacheImpl) buildUsersPermissions() (usersPermissions, error) {
 	for _, usr := range users {
 		for _, roleBinding := range roleBindings {
 			if roleBinding.Spec.Has(v1.KindUser, usr.Metadata.Name) {
-				rolePermissions := findRole(roles, roleBinding.Metadata.Project, roleBinding.Spec.Role).Spec.Permissions
+				projectRole := findRole(roles, roleBinding.Metadata.Project, roleBinding.Spec.Role)
+				if projectRole == nil {
+					logrus.Warningf("role %q listed in the role biding %s/%s does not exist", roleBinding.Spec.Role, roleBinding.Metadata.Name, roleBinding.Metadata.Project)
```

Comment:
It will be bettet liek that, no?
```suggestion
					logrus.Warningf("role %q listed in the role biding %s/%s does not exist", roleBinding.Spec.Role, roleBinding.Metadata.Project, roleBinding.Metadata.Name)
```